livecheck/main.py

```diff
@@ -393,6 +399,10 @@ def do_main(*, auto_update: bool, keep_old: bool, cat: str, ebuild_version: str,
                 update_dotnet_ebuild(new_filename, settings.dotnet_projects[cp], cp)
             elif cp in settings.jetbrains_packages:
                 update_jetbrains_ebuild(new_filename, url)
+            if git:
+                sp.run(('/usr/bin/ebuild', new_filename, 'digest'), check=True)
```

Comment:
I think we should either use it without the absolute path or use `"{os.environ.get('EPREFIX', '')}/usr/bin/ebuild"`. This goes for pkgdev too.